packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -488,6 +487,12 @@ export class StripeHelper extends StripeHelperBase {
       payment_provider: 'stripe',
     });
 
+    const idempotencyParams = [customerId, priceId];
+    if (paymentMethod?.card?.fingerprint) {
+      idempotencyParams.push(paymentMethod?.card?.fingerprint);
+    }
```

Comment:
You could just include it, if its not there it'll be undefined, so its the same as not passing it.